Scheduling a navigation to a Blob URL should keep the URL alive until the navigation actually occurs

cdumez opened the pull request:
#### 568ffc6e49bac0233128a6f4ef332868d4d58712
<pre>
Scheduling a navigation to a Blob URL should keep the URL alive until the navigation actually occurs
<a href="https://bugs.webkit.org/show_bug.cgi?id=243936">https://bugs.webkit.org/show_bug.cgi?id=243936</a>

Reviewed by Geoffrey Garen.

* LayoutTests/imported/w3c/web-platform-tests/FileAPI/url/url-in-tags-revoke.window-expected.txt:
* Source/WebCore/loader/NavigationScheduler.cpp:

Canonical link: <a href="https://commits.webkit.org/253435@main">https://commits.webkit.org/253435@main</a>
</pre>

geoffreygaren commented on Source/WebCore/loader/NavigationScheduler.cpp:
```diff
@@ -163,6 +165,7 @@ class ScheduledURLNavigation : public ScheduledNavigation {
     Ref<Document> m_initiatingDocument;
     RefPtr<SecurityOrigin> m_securityOrigin;
     URL m_url;
+    BlobURLHandle m_blobURLLifetimeExtender;
```
I kind of wish this behavior were automatic to URL. But that's a bigger change.

geoffreygaren reviewed (approved):
r=me

cdumez committed:
Scheduling a navigation to a Blob URL should keep the URL alive until the navigation actually occurs
https://bugs.webkit.org/show_bug.cgi?id=243936

Reviewed by Geoffrey Garen.

* LayoutTests/imported/w3c/web-platform-tests/FileAPI/url/url-in-tags-revoke.window-expected.txt:
* Source/WebCore/loader/NavigationScheduler.cpp:

Canonical link: https://commits.webkit.org/253435@main
